coral/src/app/features/approvals/schemas/components/SchemaRequestDetails.tsx

```diff
@@ -0,0 +1,79 @@
+import { SchemaRequest } from "src/domain/schema-request";
+import { Flexbox, Grid, GridItem, StatusChip } from "@aivenio/aquarium";
+import MonacoEditor from "@monaco-editor/react";
+
+type DetailsModalContentProps = {
+  request?: SchemaRequest;
+};
+
+const Label = ({ children }: { children: React.ReactNode }) => (
+  <dt className="inline-block mb-2 typography-small-strong text-grey-60">
+    {children}
+  </dt>
+);
+function SchemaRequestDetails(props: DetailsModalContentProps) {
+  const { request } = props;
+  if (!request) return null;
+  return (
+    <Grid htmlTag={"dl"} cols={"2"} rowGap={"6"}>
+      <Flexbox direction={"column"}>
+        <Label>Environment</Label>
+        <dd>
+          <StatusChip text={request.environmentName} />
+        </dd>
+      </Flexbox>
+      <Flexbox direction={"column"}>
+        <Label>Topic name</Label>
+        <dd>{request.topicname}</dd>
+      </Flexbox>
+
+      <GridItem colSpan={"span-2"}>
+        <Flexbox direction={"column"}>
+          {/*//@TODO add editor preview*/}
+          <Label>Schema</Label>
+          <dd>
+            <MonacoEditor
+              data-testid="topic-schema"
+              height="300px"
+              language="json"
+              theme={"light"}
+              value={request.schemafull}
+              loading={"Loading preview"}
+              options={{
+                ariaLabel: "Schema preview",
+                readOnly: true,
+                domReadOnly: true,
+                renderControlCharacters: false,
+                minimap: { enabled: false },
+                folding: false,
+                lineNumbers: "off",
+                scrollBeyondLastLine: false,
+                scrollbar: {
+                  vertical: "hidden",
```

Comment:
I think the vertical bar should not be hidden in this case, as the content may be scrollable, and seeing the bar might help to understand that?

Reply:
The content is scrollable, in both directions! I understood that this is describing if the scrollbar is visible also when the user / mouse is not in the editor (it's not the same behaviour as in CSS). But I just checked and we actually don't need that config in our readonly mode, it's behaving as it should without setting that!